hw/lint/Makefile

```diff
@@ -0,0 +1,103 @@
+# Copyright lowRISC contributors.
```

Comment:
I'd like to get rid of this wrapper and put the functionality into the core files as post-run script. That can come in a follow-up, however.

Reply:
Ok I am happy to try this out in a follow-up (haven't looked at the post-run script functionality in fusesoc yet).